src/renderer/components/__tests__/cronjob.store.test.ts

```diff
@@ -66,12 +88,42 @@ const otherSuspendedCronJob = new CronJob({
     namespace: "default",
     selfLink: "/apis/batch/v1beta1/cronjobs/default/otherSuspendedCronJob",
   },
-  spec,
+  spec: {
+    schedule: "test",
+    concurrencyPolicy: "test",
+    suspend: true,
+    jobTemplate: {
+      metadata: {},
+      spec: {
+        template: {
+          metadata: {},
+          spec: {
+            containers: [],
+            restartPolicy: "restart",
+            terminationGracePeriodSeconds: 1,
+            dnsPolicy: "no",
+            hostPID: true,
+            schedulerName: "string",
+          },
+        },
+      },
+    },
+    successfulJobsHistoryLimit: 1,
+    failedJobsHistoryLimit: 1,
+  },
 });
 
-scheduledCronJob.spec.suspend = false;
-
 describe("CronJob Store tests", () => {
+  let cronJobStore: CronJobStore;
+
+  beforeEach(() => {
+    const di = getDiForUnitTesting({ doGeneralOverrides: true });
+
+    di.override(createStoresAndApisInjectable, () => true);
```

Comment:
What does the boolean return value mean?

Reply:
It means that all stores and apis instances should be able to be created (as if in cluster frame). I uses the specific name of `createStoresAndApis` to make this clear.

Reply:
But this should be apparent from naming :)

Reply:
> Does "createStoresAndApis" not mean that for you?

No, and it obviously shouldn't.

This is **easy**: For the sake of apparent naming:

Instead of boolean, one could return `{ storesAndApisAreUsable: boolean }`. Or better yet there could be `storesAndApisAreUsableInjectable`.

It is better to have all stuff like this as apparent in naming as possible, everywhere.

This is **hard**: now that the apparent naming gives us `storesAndApisAreUsable`, this communicates a severe flaw in design, as getting information like that should not require attempt to create stores and API's first. Also a better design would not permit a scenario where there is possibility where `storesAndApisAreUsable` is unexpectedly false. If defensive code is the only way to achieve this, how about making `instantiate` of `createStoresAndApisInjectable` blow up if `storesAndApisAreUsable` is false?